tests/conftest.py

```diff
@@ -194,9 +194,11 @@ def pytest_addoption(parser):
 
 def initial_db_init():
     from airflow.utils import db
+    from airflow.www.app import create_app
 
     db.resetdb()
     db.bootstrap_dagbag()
+    create_app(config={'UPDATE_FAB_PERMS': True})
```

Comment:
This change seems out of place -- doing this here is going to hugely slow down running many tests that don't need or care about FAB.


What is this change for?

Reply:
This is to create the permissions. Before now, it was done through `airflow db upgrade`, at this migration: https://github.com/apache/airflow/blob/e2fd41f7b14adef2c3a88dde14d088b5ef93b460/airflow/migrations/versions/0084_2_1_0_resource_based_permissions_for_default_.py#L142 whenever we run tests.
Looking at it now,  'UPDATE_FAB_PERMS' is set to False